src/views/Invite/InviteEmail.js

```diff
@@ -187,6 +187,27 @@ export default function InviteEmail() {
       throw new Error('Internal Error: Missing Contracts/Web3/Wallet');
     }
 
+    //TODO want to do this on-input, but that gets weird. see #188
+    let errorCount = 0;
+    const mailsVerified = inputs.map(async input => {
```

Comment:
nothing is returned from this map — is mailsVerified used? we probbaly want either a `reduce` to collect the number of errors or just a normal for loop for iterating the inputs

Reply:
Is this not also an ES6-acceptable way of "do x for every item in array"? `mailsVerified` holds the promises, right?